.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+**Description:**
+
+(Describe the high level scope of new or changed features)
+
+**Workflow steps:**
+
+(How does one use this feature, and how has it changed)
+
+**Documentation links affected:**
+
+(list any documentation links that you created, or existing ones that you've identified as needing updates, along with a brief description)
```

Comment:
I am putting together a doc about docs here -- https://docs.google.com/document/d/1SRC9VS9zyCzWl3n4HXHbc4wPB1eLxJHkA2rtu9ZNokM/edit?usp=sharing. It's not done but might be useful to link in the template.